app/code/Magento/InventoryInStorePickupApi/Api/Data/PickupLocationInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupApi\Api\Data;
+
+/**
+ * @TODO Replace with autogenerated immutable realization.
```

Comment:
👍 
Makes sense to create a ticket and link it here for the future reference.

Reply:
Done: #2126

Reply:
Now this `@todo` may also include a link to magento-engcom/msi#2126

When we come back to this class in a half a year this would be extremely helpful to find this note.

Reply:
It just in line under:
` @see Please check issue for more details: https://github.com/magento-engcom/msi/issues/2126`
:)

Reply:
Too bad I was blind when doing review previous time :)